man/machinectl.xml

```diff
@@ -373,8 +373,15 @@
         <para>To interactively start a container on the command line
         with full access to the container's console, please invoke
         <command>systemd-nspawn</command> directly. To stop a running
-        container use <command>machinectl poweroff</command>, see
-        below.</para></listitem>
+        container use <command>machinectl poweroff</command> or
+        <command>machinectl stop</command>, see below.</para></listitem>
+      </varlistentry>
```

Comment:
I think we should only point people to "machinectl poweroff" here, as before, but mention there that "stop" is a convenience alias.